src/course/handbook/asking-for-help.md

```diff
@@ -0,0 +1,61 @@
+# Asking for help
+
+Learning how to ask for help effectively is important. Every developer needs help at some point, especially when they're learning new things. You can make it easier for people to help you, and increase the likelihood of solving your problem.
+
+## Quick summary
+
+Here are the steps to follow when asking for help. More detail is provided below!
+
+1. Try to debug on your own
+1. Google the problem
+1. Describe the problem and what you've tried
+1. Make sure to include a code sample
+1. Update with the solution
+
+## When to ask
+
+Make sure you've tried to solve the problem on your own first. Often it can be as simple as a typo, or forgetting to save your changes—it's best to ensure you _really_ have a problem before involving someone else. Make sure you have at least googled your problem!
+
+The amount of time you spend trying to solve the problem should be proportional to how much time you have. If you're doing a 1 hour workshop then spending 15 minutes stuck is a bad use of your time. You should probably ask for help after 5 minutes. However if you're on a 2 day project you should spend more like 20 minutes searching for your own solution.
+
+It's important to develop your problem-solving and debugging skills. If more experienced people always tell you what's wrong then you'll struggle later in your career. Read the [debugging guide](../debugging/) for more information on solving code problems.
+
+## How to ask
+
+Once you're sure you have a problem you cannot quickly solve on your own you should post a message asking for help in the appropriate channel. This could be your cohort's `#help-and-solutions` Discord channel, or the more general `#facoverflow`, or even somewhere public like Stack Overflow or a project's GitHub Discussions.
+
+### Describe the problem
+
+A good message should describe the problem, and especially what you want to happen. Often people describe a specific syntax issue, when really their problem is with their whole approach. For example for "how can I run these three functions one after the other?" the most helpful solution might be "do it all in a single SQL query instead".
```

Comment:
Maybe we should pick an example more close to the fundamentals so this guide can be used across all stages of FAC?

Reply:
Good point, I was just struggling to think of one. Any better ideas? 😅 

"How can I swap the date and month in this date string?" -> "You shouldn't, you should use the built-in Internationalization API"